device/device.go

```diff
@@ -136,7 +136,7 @@ func (d *DeviceManager) GetCurrentArtifactName() (string, error) {
 			log.Errorf("Could not read artifact name from database: %s", err.Error())
 		}
 	}
-	log.Debugf("Returning artifact name from %s file.", d.ArtifactInfoFile)
+	log.Infof("Returning artifact name from %s file.", d.ArtifactInfoFile)
```

Comment:
I would say this deserves a warning, actually.

Reply:
Yes, unless it is having issues with the database of course.